css/style.css

```diff
@@ -65,7 +65,43 @@ body {
   left: 10px;
 }
 
-/*end comodities*/
+.pointer-list {
+  border-radius: 50%;
+  display: block;
+  margin: 10px;
+  list-style-position: initial;
+  list-style-type: disc;
+}
+
+.pointer-list__point {
+  border-radius: 50%;
+  padding-left: 0%;
+  margin-left: 14.5px;
+  margin-right: 14.5px;
+  display: inline-block;
+}
+.oval{
+  width: 14px;
+  height: 14px;
+}
+.oval-gray{
```

Comment:
BEM.... `ponter-list__pointer--gray` seems like the correct name, same for black